### PR TITLE
Update spanish.txt

### PR DESCRIPTION
**Pull request by headshot2017**

### CODE REVIEW

**Comment by def-** [comment]
@edg-l Can you take a look please?

File: data/languages/spanish.txt [hunk 1]
```diff
@@ -1160,131 +1161,131 @@ Grabs
...
-== 
+== Este jugador ya existe
 
 Your nickname '%s' is already used (%d points). Do you still want to use it?
-== 
+== Tu nombre '%s' ya está usado (%d puntos). ¿Deseas seguir usándolo?
 
 Checking for existing player with your name
-== 
+== Comprobando si ya hay un jugador con tu nombre
```
Comment (edg-l) [inline data/languages/spanish.txt:1185]: ya hay -> existe

maybe

**Review by edg-l — COMMENTED** [review]
looks good

File: data/languages/spanish.txt (hunk 1 above)
Comment (headshot2017) [reply data/languages/spanish.txt]: done

**Pull request merged**